src/components/data_search/DatasetSearchTable.jsx

```diff
@@ -180,222 +165,21 @@ export const DatasetSearchTable = (props) => {
-                      const primaryDataUse = map(dataUse?.primary, 'code').join(', ');
-                      const secondaryDataUse = map(dataUse?.secondary, 'code').join(', ');
-                      const mergedDataUse = compact(concat(primaryDataUse, secondaryDataUse)).join(', ');
-                      return <a
-                        id={`${index}_linkTranslatedDul`}
-                        name="link_translatedDul"
-                        onClick={()=>openTranslatedDUL(dataUse)}
-                      >{mergedDataUse}</a>;
-                    }
-                  },
-                  {
-                    value: dataset.dataTypes,
-                  },
-                  {
-                    value: dataset.participantCount,
-                  },
-                  {
-                    value: () => {
-                      let accessType;
-                      if (dataset.accessManagement === 'external') {
-                        accessType = dataset.url ? <Link href={dataset.url}>External to DUOS</Link> : 'External to DUOS';
-                      } else if (dataset.accessManagement === 'open') {
-                        accessType = dataset.url ? <Link href={dataset.url}>Open Access</Link> : 'Open Access';
-                      } else {
-                        accessType = dataset.dac?.dacEmail ? <Link href={'mailto:' + dataset.dac.dacEmail}>{dataset.dac?.dacName}</Link> : dataset.dac?.dacName;
-                      }
-                      return accessType;
-                    }
-                  },
-                  {
-                    value: () => {
-                      const exportableSnapshots = exportableDatasets[dataset.datasetIdentifier] || [];
-                      if (exportableSnapshots.length === 0) {
-                        return dataset.dataLocation;
-                      }
-                      return exportableSnapshots.map((snapshot, i) =>
-                        <Link
-                          key={`${i}`}
-                          href={`${tdrApiUrl}/snapshots/${snapshot.id}`}
-                          target="_blank"
-                        >
-                          {snapshot.name}{'\n'}
-                        </Link>);
-                    }
-                  },
-                  {
-                    value: () => {
-                      const exportableSnapshots = exportableDatasets[dataset.datasetIdentifier] || [];
-                      return exportableSnapshots
-                        .map((snapshot, i) =>
-                          <DatasetExportButton
-                            key={`${i}`}
-                            snapshot={snapshot}
-                            title={`Export snapshot ${snapshot.name}`} />);
-                    }
-                  },
-                ],
-              };
-            }),
-          }
-        };
-      }),
-    };
-
-    setTableData(table);
-  }, [filtered, exportableDatasets, tdrApiUrl]);
+  }, [filtered]);
```

Comment:
I think we may have lost the export to TDR functionality here. Note, for example, that we're not getting the `tdrApiUrl` anywhere else in this PR.